tests/unit/executors/test_decorators.py

```diff
@@ -9,6 +9,8 @@
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 
 
+# TODO add test empty index with search
```

Comment:
this TODO should not be here right?

Reply:
the tests would belong somewhere else I think?

Reply:
A very basic version is here https://github.com/jina-ai/jina/pull/1340/files#diff-f72cb602d29fda8028f46b7b7d9940710c62d5b7fb95966f1fa58f6c924cc2d2R242